tools/create_service_diagnostics_bundle.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python3
+
+# Dependencies:
+# - DC/OS CLI
+# - jq
+
+# Notes:
+# - When run in the dcos-commons docker image the CLI version is specified in the test_requirements.txt
+#   file.
+
+from typing import Any
+import argparse
+import json
+import logging
+import sys
+
+from diagnostics import FullBundle
+import sdk_cmd
+
+log = logging.getLogger(__name__)
+
+
+def current_cluster_name() -> (bool, str):
+    rc, stdout, stderr = sdk_cmd.run_raw_cli("config show cluster.name", print_output=False)
+
+    if rc != 0:
+        if "Property 'cluster.name' doesn't exist" in stderr:
+            return (
+                False,
+                "No cluster is set up. Please run `dcos cluster setup`\nstdout: '{}'\nstderr: '{}'".format(
+                    stdout, stderr
+                ),
+            )
+        else:
+            return (False, "Unexpected error\nstdout: '{}'\nstderr: '{}'".format(stdout, stderr))
+
+    return (True, stdout)
+
+
+def is_authenticated_to_dcos_cluster() -> (bool, str):
+    rc, stdout, stderr = sdk_cmd.run_raw_cli("service", print_output=False)
+
+    if rc != 0:
+        (success, cluster_name_or_error) = current_cluster_name()
+
+        if "dcos auth login" or "Missing required config parameter" in stderr:
```

Comment:
This does not do what you think :-)
```
>>> if "bug" or "impossible" in "badger":
...   print("peekaboo!")
... 
peekaboo!
>>> 
```

Reply:
Good catch caaa42f3661c6650175921179b0a0e26bbd2aa0d